### PR TITLE
Documentation: Suggest using `fleetctl ssh` with `systemctl restart`

### PR DESCRIPTION
**Pull request by antrik**
Partially addresses https://github.com/coreos/fleet/issues/975

### CODE REVIEW

**Comment by tixxdz** [comment]
Thank you! lgtm.

Cc @jonboulle @joshix native speaker please ?

File: Documentation/using-the-client.md [hunk 1]
```diff
@@ -104,6 +104,17 @@ Unit goodbye.service loaded on 85c0c595.../172.17.8.102
 
 If the unit does not exist when calling `start`, fleetctl will first search for a local unit file, submit it and schedule it.
 
+### Restart units
+
+Fleet currently doesn't have a dedicated `restart` command;
```
Comment (joshix) [inline Documentation/using-the-client.md]: Replace lines 109-112 with the following, and do not wrap lines, please:
```
`fleetctl` doesn't have a `restart` subcommand. In many cases it is simple to [use the `fleetctl ssh` subcommand](#ssh-dynamically-to-host) to execute `systemctl restart` directly on the target host:
```

Comment (kayrus) [inline Documentation/using-the-client.md]: Could you also put all links references at the bottom? https://github.com/coreos/docs/blob/master/STYLE.md#marking-down-the-link

I.e.
```
[use the `fleetctl ssh` subcommand][ssh-dynamically]
... ... ...
[ssh-dynamically]: #ssh-dynamically-to-host
EOF
```

**Comment by antrik** [comment]
@kayrus note that half of the existing links in this file don't stick to that convention either...

Anyway, applied both changes now.

**Comment by antrik** [comment]
@kayrus ah, just saw that you submitted a PR for this :-)

**Pull request merged**